2.1/README.md

```diff
@@ -0,0 +1,45 @@
+HTTP 301 Moved Permanently
+--------------------------
+
+**Where are the 2.1 aspnet-docker images?**
+
+In 2.1, the ASP.NET Core Docker images have migrated to https://github.com/dotnet/dotnet-docker.
+
+**How can I upgrade from 1.x/2.0 to 2.1?**
+
+You can upgrade to 2.1 and by changing the repository name in the `FROM` line in your Dockerfile using this mapping:
+
+Current | Upgrade
+--------|--------------
+`microsoft/aspnetcore:1.0`<br>`microsoft/aspnetcore:1.1`<br>`microsoft/aspnetcore:2.0` | `microsoft/dotnet:2.1-aspnetcore-runtime`
+`microsoft/aspnetcore-build:1.0`<br>`microsoft/aspnetcore-build:1.1`<br>`microsoft/aspnetcore-build:2.0` | `microsoft/dotnet:2.1-sdk`
+
+**I was using NodeJS in `microsoft/aspnetcore-build`, but this is missing from `microsoft/dotnet:2.1-sdk`. What should I do?**
+
+You can either install NodeJS by adding a few lines of code to your Dockerfile that download and extract NodeJS, 
+or you can use the multi-stage feature of Docker and the official NodeJS images.
+
+Sample code to install NodeJS on your own:
```

Comment:
What sample do you see as the most common?  I was thinking multi-stage using the NodeJS image would be.  I think the most common sample should be listed first.

Reply:
It's sort of new territory since NodeJS has been bundled in the image until this point, so not sure which is more common. I would personally use multi-stage builds, but know lots of users have MSBuild targets invoking npm.